### PR TITLE
Issue 145-Notification always in unread status

### PR DESCRIPTION
**Pull request by ghost**
Mark notifications as read when related posts are visited. When a notification is clicked, all notifications related to the post are marked as read.

### CODE REVIEW

File: app/components/Notification/index.jsx [hunk 1]
```diff
@@ -43,7 +52,15 @@ let NotifiItem = React.createClass({
         };
         let timeStyle = {
             fontSize: "0.9em"
-        }
+        };
+        let achorReadStyle = {
+            color: "#aaa",
+            cursor: "pointer"
```
Comment (LeonLiuY) [inline app/components/Notification/index.jsx:59]: We need to follow material design style guideline:
http://www.google.co.jp/design/spec/style/color.html#color-ui-color-application

And we want to enable theme support so we should use theme variable:
`
this.context.muiTheme.palette.[colorName]
`

Comment (ghost) [inline app/components/Notification/index.jsx:59]: Sorry, I forget this.

File: app/components/Notification/index.jsx [hunk 2]
```diff
@@ -129,7 +145,7 @@ let Notification = React.createClass({
                                   iconClassName={iconClassName}/>;
         let menu = this.state.notifications.map(n => {
             let sender = UserStore.getUser(n.from_user_id);
-            return <NotifiItem key={n.id} sender={sender} time={n.created_at} message={n.content} read={n.checked}/>;
+            return <NotifiItem id={n.id} url={n.url} sender={sender} time={n.created_at} message={n.content} read={n.checked}/>;
```
Comment (LeonLiuY) [inline app/components/Notification/index.jsx]: Don't remove `key` property, it is required by React.
https://facebook.github.io/react/docs/create-fragment.html

Comment (ghost) [inline app/components/Notification/index.jsx]: OK

**Pull request merged**